sql/2019/17_CDN/17_03.sql

```diff
@@ -0,0 +1,22 @@
+#standardSQL
+# 07_03: Percentiles of TTFB for CDN
+SELECT
+  client,
```

Comment:
I think we want to see the TTFB _per CDN_ not over all CDN requests.
```suggestion
  client,
  cdn,
```

Reply:
Corrected